File: Java/Binary.java
import java.util.Scanner;
public class Binary {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n= sc.nextInt();
        int[] arr= new int[20];
        int i=0;
        while(n!=0){
            int r=n%2;
            arr[i]=r;
            i++;
            n=n/2;
        }

        for(int j=0;j<i;j++){
            System.out.print(arr[i-j-1]);
        }
    }
    
}

File: C/Binary.c
#include <stdio.h>
//char A[4];
//A[3]='\0';

void Binary(int n,char* A)
{
    if(n<1)
    {
        
        printf("%s\n",A);
    }
    else
    {
        A[n-1]=1;
        Binary(n-1,A);
        A[n-1]=0;
        Binary(n-1,A);

  
        }    
    }

void main()
{
    char B[5];
    B[4]='\0';
    Binary(3,B);
}